cds_migrator_kit/cli.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# cds-migrator-kit is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+"""cds-migrator-kit command line module."""
+
+import importlib
+
+import click
+
+
+def import_module(module_name):
```

Comment:
That is redefined here. Could you maybe split it in a common file and reuse it across the repository?